cmd/secretonepassword.go

```diff
@@ -39,12 +39,27 @@ func (c *Config) runOnepasswordCmd(cmd *cobra.Command, args []string) error {
 	return c.run("", c.Onepassword.Command, args...)
 }
 
-func (c *Config) onepasswordFunc(item string) interface{} {
-	if data, ok := onepasswordCache[item]; ok {
+func (c *Config) onepasswordFunc(args ...string) interface{} {
+	if len(args) < 1 || len(args) > 2 {
+		panic(fmt.Errorf("function takes an item ID and optionally a vault ID"))
```

Comment:
Sure, I'm happy to make these changes :smile:

I actually deleted the `onepassword:` prefix because it was already being added by whatever ultimately prints out the message.  Before I deleted it, this was the message I was getting:
`chezmoi: template: C:\Users\zb\.local\share\chezmoi\1pass-test.tmpl:2:16: executing "C:\\Users\\zb\\.local\\share\\chezmoi\\1pass-test.tmpl" at <onepassword "uuid" "vault" "foo" "bar" "baz">: error calling onepassword: onepassword: function takes an item ID and optionally a vault ID`

But I totally get not wanting to rely on that behavior, especially since it's not obvious from the call site that it will happen.